daemon/src/model/cfd.rs

```diff
@@ -363,6 +364,192 @@ impl Cfd {
 
     #[allow(dead_code)]
     pub const CET_TIMELOCK: u32 = 12;
+
+    pub fn transition_to(&self, event: CfdStateChangeEvent) -> Result<CfdState> {
+        use CfdState::*;
+
+        // TODO: Display impl
+        tracing::info!("Cfd state change event {:?}", event);
+
+        let order_id = self.order.id;
+
+        let new_state = match event {
```

Comment:
The idea was to prepare this so that **all** state transitions would be handle in one function (eventually). 
Granted, there is only one type of events at the moment, but I think it's OK to outline this. 
If the indentation is annoying we can also match on the complete states instead of indenting. Will keep that for now if that's OK.